torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1052,6 +1052,37 @@ class IrParser {
           nullptr);
     }
 
+    { // LTC uses threshold_backward for relu_backward
+      auto ptr_op = getOperatorForLiteral(
+          "aten::threshold_backward(Tensor grad_output, Tensor self, Scalar threshold) -> Tensor");
+      REGISTER_PARSE_RULE(
+          ptr_op,
+          {
+            MemoryFormat format;
+            std::list<Val*> list_val;
+            std::tie(format, list_val) = getConsistentValues(
```

Comment:
nitpick: we should be able to easily handle permutation for pw operations.
e.g.
https://github.com/csarofeen/pytorch/blob/ddaa2a5535c5f44af5f8c21682a41489e149e1d7/torch/csrc/jit/codegen/cuda/parser.cpp#L710-L713
https://github.com/csarofeen/pytorch/blob/ddaa2a5535c5f44af5f8c21682a41489e149e1d7/torch/csrc/jit/codegen/cuda/parser.cpp#L727-L728

Reply:
Fixed.